_posts/2019-09-10-unbearable-lightness-of-refactoring.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: epic
+title: "Unbearable Lightness of Refactoring"
+date: "2019-09-10"
+author: [ashkan]
+categories: [refactoring, community, culture]
+comment_id: 522
+---
+
+As engineers we are constantly in the process of building new features and improve our existing ones. Nowadays, with the help of tools and processes like code reviews one could argue the quality of the codes being written has risen. At Artsy a pull request normally has one Assignee and possibly one or more Reviewers, so why we still do lot of refactoring?
```

Comment:
```suggestion
As engineers we are constantly in the process of building new features and improving our existing ones. Nowadays, with the help of tools and processes like code reviews one could argue the quality of the code being written has risen. At Artsy a pull request normally has one Assignee and possibly one or more Reviewers, so why we still do lot of refactoring?
```

Reply:
I think this section should be reworked. 

I see what you're getting at... even though we're continually writing code, we use tools like code reviews to ensure we're outputting high quality code. We still have to do a lot of refactoring. Why?

The issue, I think, is that it seems like the blog post will address this issue specifically... why do we need to do refactoring when we should be producing high quality code? 

Maybe instead you could write about how `quality worth of art` is one of our company principles and that refactoring is a good mechanism to achieve that.